frontend/src/metabase/core/components/MarkdownPreview/MarkdownPreview.styled.tsx

```diff
@@ -10,3 +12,10 @@ export const TruncatedMarkdown = styled(Markdown)`
   overflow-wrap: break-word;
   white-space: pre-line;
 `;
+
+export const TooltipMarkdown = styled(Markdown)`
+  hr {
```

Comment:
I think we should pass something to `Markdown` component to make it style this differently. To consolidate all markdown styles in one place

Reply:
Nice catch :+1: There were other places that needed this.

 adea9aa15da4adf33fa874ca0fca366714ec152d